packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -219,8 +219,8 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
             assert(this.state === SummaryManagerState.Starting, 0x263 /* "Expected: starting" */);
             this.state = SummaryManagerState.Running;
 
-            summarizer.on("summarizingError",
-                (warning: ISummarizingWarning) => this.emit("summarizerWarning", warning));
+            this.logger.sendErrorEvent({ eventName: "summarizingError" },
+                (warning: string) => createSummarizingWarning(warning, true));
```

Comment:
This doesn't look right to me...

Reply:
Agreed, it looks like when you deleted the emit in summarizer, then you didn't need any of this code.